tests/acceptance/features/apiSettings/notificationSetting.feature

```diff
@@ -607,3 +607,24 @@ Feature: Notification Settings
       | message                                   |
       | Alice Hansen added you to Space new-space |
     But user "Brian" should not have a notification related to space "new-space" with subject "Space disabled"
+
+
+  Scenario: check share expired in-app and mail notification for Personal space resource
```

Comment:
this scenarios should be in `apiNotification` suite  as `shareExpireNotification.feature`